### PR TITLE
P3A - Differentiate between 0 and 1 window

### PR DESCRIPTION
**Pull request by bsclifton**
We suspect there are people who have Brave open with no windows open when this question runs.

Fixes https://github.com/brave/brave-browser/issues/9676

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [x] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [x] Windows
- [x] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

**Comment by bsclifton** [comment]
NOTE: will need to update docs when this is changed:
https://github.com/brave/brave-browser/wiki/P3A

**Comment by bsclifton** [comment]
CI looks good! There was a known intermittent browser test (captured with https://github.com/brave/brave-browser/issues/9339) which failed on macOS:
```
16:19:20  1 test failed:
16:19:20      BraveRewardsBrowserTest.NotVerifiedWallet (../../brave/components/brave_rewards/browser/rewards_service_browsertest.cc:2356)
```
https://ci.brave.com/job/pr-brave-browser-bsc-p3a-window-count/1/execution/node/499/log/

Travis-CI did have a problem with pep8; I captured this with https://github.com/brave/brave-browser/issues/9732

**Comment by bsclifton** [comment]
Updated https://github.com/brave/brave-browser/wiki/P3A 👍

File: components/p3a/brave_histogram_rewrite.cc [hunk 1]
```diff
@@ -88,15 +88,17 @@ void DoHistogramBravezation(base::StringPiece histogram_name,
...
       answer = 1;
-    } else {
+    } else if (2 <= sample && sample <= 5) {
       answer = 2;
+    } else {
+      answer = 3;
     }
 
-    UMA_HISTOGRAM_EXACT_LINEAR("Brave.Core.WindowCount", answer, 2);
+    UMA_HISTOGRAM_EXACT_LINEAR("Brave.Core.WindowCount.2", answer, 2);
```
Comment (iefremov) [inline components/p3a/brave_histogram_rewrite.cc:101]: max value should be changed to 3
`UMA_HISTOGRAM_EXACT_LINEAR("Brave.Core.WindowCount.2", answer, 3);`

Comment (bsclifton) [reply components/p3a/brave_histogram_rewrite.cc:101]: Updated here https://github.com/brave/brave-core/pull/5528
😄

**Comment by GeetaSarvadnya** [comment]
Verification passed on _Windows 10 x64_ using

```
Brave | 1.10.53 Chromium: 81.0.4044.138 (Official Build) nightly (64-bit)
-- | --
Revision | 8c6c7ba89cc9453625af54f11fd83179e23450fa-refs/branch-heads/4044@{#999}
OS | Windows 10 OS Version 1803 (Build 17134.1006)
```

**Pull request merged**